Classes/DDOSLogger.h

```diff
@@ -36,3 +36,7 @@ API_AVAILABLE(macos(10.12), ios(10.0), watchos(3.0), tvos(10.0))
 @property (class, readonly, strong) DDOSLogger *sharedInstance;
 
 @end
+
+@interface DDOSLogger (Variations)
+- (instancetype)initWithSubsystem:(NSString *)subsystem category:(NSString *)category;
```

Comment:
Since this should become the designated initializer, it can also go into the main `@interface`.